pkg/controller/ads/ads_processor.go

```diff
@@ -187,26 +189,22 @@ func (p *processor) handleCdsResponse(resp *service_discovery_v3.DiscoveryRespon
 }
 
 func (p *processor) handleEdsResponse(resp *service_discovery_v3.DiscoveryResponse) error {
-	var (
-		err            error
-		loadAssignment = &config_endpoint_v3.ClusterLoadAssignment{}
-	)
-
+	var loadAssignment = &config_endpoint_v3.ClusterLoadAssignment{}
 	p.lastNonce.edsNonce = resp.Nonce
-	current := sets.New[string]()
 	for _, resource := range resp.GetResources() {
-		if err = anypb.UnmarshalTo(resource, loadAssignment, proto.UnmarshalOptions{}); err != nil {
+		if err := anypb.UnmarshalTo(resource, loadAssignment, proto.UnmarshalOptions{}); err != nil {
 			continue
 		}
 		cluster := p.Cache.ClusterCache.GetApiCluster(loadAssignment.GetClusterName())
 		// fix exceptional scenarios: receive eds push after cds has been deleted
 		if cluster == nil {
+			fmt.Println("------not found ")
```

Comment:
```suggestion
			fmt.Printf("Cluster %s not found in cluster cache\n", loadAssignment.GetClusterName())
```
I think this log is much clearer

Reply:
I meant to remove it